src/engine/ruleset/schemas/wazuh-logpar-types.json

```
@@ -1119,6 +1106,7 @@
         "wazuh.origin": "keyword",
         "wazuh.registered_ip": "keyword",
         "wazuh.source": "keyword",
+        "wazuh.dns.question.original": "text",
```

Change field `wazuh.dns.question.original` to `suricata.dns.question`
```suggestion
        "suricata.dns.question.original": "text",
```

---

this information already exist in dns.question.name